### PR TITLE
umask can also be set via bashScriptExtraDefines

### PR DESCRIPTION
**Pull request by mkurz**
I tried to set the umask for a app running via docker. The only way to do so is to make use of `bashScriptExtraDefines`. Of course this also does not only work for docker, but whenever you run an app with Linux.

### CODE REVIEW

File: src/sphinx/recipes/index.rst [hunk 1]
```diff
@@ -17,5 +17,6 @@ This section provides recipes for common configurations. If you can't find what
    scalajs
    package_configuration
    embedded-jvm
+   umask
```
Comment (mkurz) [inline src/sphinx/recipes/index.rst]: The umask page was not linked yet...

File: src/sphinx/archetypes/cheatsheet.rst [hunk 2]
```diff
@@ -98,7 +98,7 @@ For a bash script this could look like this.
         """addJava "-Xmx2048m""""
      )
 
-For information take a look at the :doc:` customize section for java apps </archetypes/java_app/customize>`
+For more information take a look at the :ref:`according documentation<add-code-to-the-start-scripts>`.
```
Comment (mkurz) [inline src/sphinx/archetypes/cheatsheet.rst]: This "link" is currently broken. Fixed it.

File: src/sphinx/recipes/umask.rst [hunk 3]
```diff
@@ -3,18 +3,36 @@
...
+your application that is running either via a System V init script or systemd, the umask can be specified via
+a custom ``/etc/default/appname`` file.
+
+The contents of this file can be specified by creating a template at ``src/templates/etc-default`` and adding a line such as the following:
 
 
 .. code-block:: bash
 
-    umask 022
+    umask 077
```
Comment (mkurz) [inline src/sphinx/recipes/umask.rst]: I changed the example to 077, because 022 is the default in every distro anyway AFAIK.

**Comment by muuki88** [comment]
Thanks for adding your findings to the documentation :heart: and clean up a few missing things :+1:

**Pull request merged**